cfpb/consumerfinance.gov
# Django url to pg env vars

In order to set us up for success in moving our database related env vars we are changing the old way of getting the database URL which was mostly hard coding the value to using the standard Postgresql Env Vars. 
---


## Additions

-Addition of PG Env Vars like PGUSER PGPASSWORD PGDATABASE PGHOST PGPORT
-Django database is now using the PG engine to connect to the DB


## Removals

- removing the dj_database_url library from our codebase 


## Changes

-


## How to test this PR

1. Pull the PR and run the helm install2. 
2. Jump into the python pod and 
3. print out the env vars in the shell, with `env | grep PG` to see all the DB related env vars added
4. you can also prove the DB connection is working via the Env Vars by running `psql "postgres://${PGUSER}:${PGPASSWORD}@${PGHOST}:${PGPORT}/${PGDATABASE}"`

## Screenshots


## Notes and todos

-


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
